server/app.js

```diff
@@ -61,7 +61,8 @@ server.listen(port, hostName, function () {
 app.post("/api/newboard", (req, res) => {
     //if user did not input their own dictionary, generate new board using default dict
     if (!req.body.customizedDict) {
-        res.json(Board.newBoard());
+        currentBoard = Board.newBoard();
```

Comment:
@GonzaloAllenPerez444 we changed the backend to do this. Don't know if this is what is required???